client/my-sites/customer-home/cards/tasks/podcasting/index.jsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Task from 'calypso/my-sites/customer-home/cards/tasks/task';
+import { TASK_PODCASTING } from 'calypso/my-sites/customer-home/cards/constants';
+import podcastingIllustration from 'calypso/assets/images/customer-home/illustration--task-podcasting.svg';
+
+const Podcasting = () => {
+	const translate = useTranslate();
+
+	return (
+		<Task
+			title={ translate( 'Grow your audience with a podcast' ) }
+			description={ translate(
+				`Easily turn your blog into a podcast with Anchor — the world's biggest podcasting platform.`
+			) }
+			actionText={ translate( 'Get started' ) }
+			// TODO replace with more appropriate URL: https://github.com/Automattic/dotcom-manage/issues/320
+			actionUrl={ `https://anchor.fm` }
+			completeOnStart={ false }
```

Comment:
One more nitpick: `completeOnStart` defaults to false so we can remove this altogether.

https://github.com/Automattic/wp-calypso/blob/5da115c261edabdb231326acaad57d6147721f88/client/my-sites/customer-home/cards/tasks/task.jsx#L40